treeherder/client/thclient/perfherder.py

```diff
@@ -0,0 +1,105 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from .client import TreeherderClient, TreeherderClientError
+
+
+class TimeInterval(object):
+    DAY = 86400
+    WEEK = 604800
+    TWO_WEEKS = 1209600
+    SIXTY_DAYS = 5184000
+    NINETY_DAYS = 7776000
+
+
+class SignatureList(object):
```

Comment:
This looks more like a dict than a list.
I would suggest to make this class inherit from dict. If you do that you can probably remove __init__, __len__ and __getitem__.